lib/src/sql/value/value.rs

```diff
@@ -1708,6 +1708,7 @@ impl Value {
 
 	/// Checks whether this value is a static value
 	pub(crate) fn is_static(&self) -> bool {
+		// TODO: are `Bytes` static?
```

Comment:
Yes `Bytes` are static. That is, they don't need to be computed to return a computed `Value`.

Reply:
Cool!